rust/web_proof/src/lib.rs

```diff
@@ -10,4 +10,4 @@ mod request_transcript;
 mod response_transcript;
 mod transcript_parser;
 mod utils;
-mod web;
+pub mod web;
```

Comment:
@marekkirejczyk your approval is unfortunately needed since I had to expose a private module in a crate that is part of the audits.